kangzj/gutenberg
# Date Range Component Prototype

## This is a WIP and not yet applied any design

PT: pejTkB-1zd-p2



## What?

- Refactor Day component into a separate file to be reused
- Created a DateRange component that works for a date range selection, which is still mostly a copy of the date picker
- Create base stories for DateRange

## Why?

Add a date range picker

## Testing Instructions

- Checkout my fork
- Choose the right node version `nvm use`
- Install `npm i`
- Run storybook `npm run storybook:dev`
- Search for `DateRange`
- Ensure the third example marks several dates as selected



### Testing Instructions for Keyboard

n/a

## Screenshots or screencast 







https://github.com/user-attachments/assets/798e3e60-8a25-48f6-bad5-fef4801ddac7
